service/src/main/resources/templates/ehr_narrative_statement_template.mustache

```diff
@@ -0,0 +1,6 @@
+<NarrativeStatement classCode="OBS" moodCode="EVN">
```

Comment:
NarrativeStatement needs to be within a component with contextConductionInd included conditional on isNested flag. See row 10 of mappings spec.